src/__test__/8bitChannelImages.read.test.ts

```diff
@@ -0,0 +1,85 @@
+import fs from 'node:fs';
+
+import { decode } from '..';
+import { BITMAPV5HEADER } from '../constants';
+
+import { testDecode } from './testDecode';
+
+const data = {
+  width: 0,
+  height: 0,
+  data: new Uint8Array(),
+  bitsPerPixel: 1,
+  colorMasks: [0x00ff0000, 0x0000ff00, 0x000000ff],
+  compression: 0,
+  channels: 1,
+  components: 1,
+  xPixelsPerMeter: BITMAPV5HEADER.DEFAULT_PIXELS_PER_METER,
+  yPixelsPerMeter: BITMAPV5HEADER.DEFAULT_PIXELS_PER_METER,
+};
```

Comment:
You should create a function which returns a new object for each test.

Each test overrides values for the next test, which may lead to unexpected things.